source/auth/tests/legacy/connection-string.yml

```diff
@@ -401,44 +401,13 @@ tests:
     mechanism: MONGODB-OIDC
     mechanism_properties:
       REQUEST_TOKEN_CALLBACK: true
```

Comment:
Can we update these tests to use `OIDC_TOKEN_CALLBACK` instead and remove the tests with the refresh token?

Reply:
Removed all tests that use callbacks from the connection string tests. They are all prose tests now.